dev-python/pytest-aiohttp/pytest-aiohttp-0.3.0-r1.ebuild

```diff
@@ -14,7 +14,6 @@ SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
 LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ppc ppc64 ~riscv sparc x86 ~x64-macos"
-IUSE=""
 
 RDEPEND="
 	>=dev-python/pytest-5.4[${PYTHON_USEDEP}]
```

Comment:
Is the indentation for the below line wrong, or is it just the editor?

Also, no tests for this?

Reply:
About indentation, I even checked using `hexdump -C` and it uses `\t` - so I guess it is Ok?

About tests - this is a very trivial package that adds `aiohttp`'s pytest support as a first class. Except of an import is does nothing.
The testsuite is tested in `aiohttp` package.